src/plugins/teleop/Teleop.cc

```diff
@@ -62,20 +68,32 @@ namespace plugins
     /// \brief Publisher.
     public: ignition::transport::Node::Publisher cmdVelPub;
 
-    /// \brief Linear velocity.
-    public: double linearVel = 0;
-    /// \brief Angular velocity.
-    public: double angularVel = 0;
+    /// \brief Forward velocity.
+    public: double maxForwardVel = 1.0;
+
+    /// \brief Vertical velocity.
+    public: double maxVerticalVel = 1.0;
+
+    /// \brief Yaw velocity.
+    public: double maxYawVel = 0.5;
+
+    /// \brief Forward scale.
```

Comment:
```suggestion
    /// \brief Forward scale, possible vales are 1 (forward), -1 (backward), or 0 (stop).
```
Below `*keyScale` comments as well

Reply:
Updated docs in [b0a7347](https://github.com/gazebosim/gz-gui/pull/440/commits/b0a7347747aa882d4ed784162b33cbdddfb5f019), let me know if it's clear